flow/util/genRuleFile.py

```diff
@@ -328,9 +332,9 @@ def gen_rule_file(design_dir, update, tighten, failing, variant, golden_metrics=
     parser.add_argument('dir', help='Path to the design directory.')
     parser.add_argument(
         '-v',
-        '--variant',
+        '--platform',
         default='base',
-        help='Flow variant [default="base"].')
+        help='Flow platform [default="asap7"].')
```

Comment:
I thought these changes had been reverted. Why are we removing `variant` and adding `platform` argument? There are other places where we use this script and it still uses the `variant` argument.